api/lib/source/polkadotV0-source.js

```diff
@@ -1066,23 +949,37 @@ class polkadotAPI {
     }
   }
 
-  getProposalParameter(proposal) {
+  async getProposalParameterDescriptionString(proposal) {
+    const api = await this.getAPI()
+
     // democracy proposals in Polkadot contain a parameter which is what is going to be changed
+    let parameterDescription = ''
     if (proposal.image) {
-      const imageProposal = JSON.parse(JSON.stringify(proposal.image.proposal))
-      if (imageProposal.args.old) {
-        return `Old: ${imageProposal.args.old} | New: ${imageProposal.args.new}`
-      } else if (imageProposal.args.code) {
-        return hexToString(imageProposal.args.code)
-      } else if (imageProposal.args.new) {
-        return String(imageProposal.args.new)
+      const {
+        meta: { documentation },
```

Comment:
I don't think this will work without calling `callIndex` first. Have you tried it yourself?

Reply:
oops, I was sleepy. OK, never mind :sweat_smile: